build.gradle

```diff
@@ -133,185 +123,124 @@ task standaloneJar(type: ShadowJar, dependsOn: [getVersion]) {
-            }
-
-            MavenPom pomJdbc = addFilter('crate-jdbc') {artifact, file ->
-                artifact.name == 'crate-jdbc'
-            }
-
-            pomJdbc.whenConfigured { pom ->
-                pom.dependencies.clear()
-                subprojects.findAll().each {
-                    if (it.hasProperty('install')) {
-                        pom.dependencies.addAll(
-                            it.install.repositories.mavenInstaller.pom.getEffectivePom().dependencies.findAll()
-                        )
-                    }
-                }
-            }
-
-            pomJdbc.project {
-                artifactId 'crate-jdbc'
-                name 'crate-jdbc'
-                description 'Crate.IO JDBC Driver'
-                url project.ext.url
-                scm project.ext.scm
+publishing {
+    publications {
+        jdbc(MavenPublication) {
+            artifactId = 'crate-jdbc'
+            artifact shadowJar
+            artifact sourcesJar
+            artifact javadocJar
+            version = project.version
+            pom {
+                name = "crate-jdbc"
+                description = "CrateDB JDBC Driver"
+                url = project.ext.url
                 licenses project.ext.licenses
                 developers project.ext.developers
-            }
-
-            MavenPom ascPom = addFilter('asc') {artifact, file -> artifact.ext == "asc" }
-            ascPom.artifactId = 'crate-jdbc'
-            ascPom.groupId = project.group
-
-            MavenPom pomJdbcStandalone = addFilter('crate-jdbc-standalone') {artifact, file ->
-                artifact.name == 'crate-jdbc-standalone'
-            }
-            pomJdbcStandalone.whenConfigured {
-                it.dependencies.clear()
-            }
-            pomJdbcStandalone.project {
-                artifactId 'crate-jdbc-standalone'
-                name 'crate-jdbc-standalone'
-                description 'Crate.IO JDBC Driver (Standalone)'
-                url project.ext.url
                 scm project.ext.scm
+            }
+        }
+        jdbcStandalone(MavenPublication) {
+            artifactId = 'crate-jdbc-standalone'
+            artifact standaloneJar
+            artifact javadocJarStandalone
+            artifact sourcesJarStandalone
+            version = project.version
+            pom {
+                name = "crate-jdbc-standalone"
+                description = "CrateDB JDBC Driver (Standalone)"
+                url = project.ext.url
                 licenses project.ext.licenses
                 developers project.ext.developers
-            }
-
-            repository(id: 'crate-jdbc', url: 'https://oss.sonatype.org/service/local/staging/deploy/maven2/') {
-                authentication(userName: project.ext.sonatypeUsername, password: project.ext.sonatypePassword)
+                scm project.ext.scm
             }
         }
     }
 }
 
+signing {
+    def signingKey = findProperty("signingKey")
```

Comment:
Could you please explain whey you mention `ORG_GRADLE_PROJECT_signingKey` in the docs, but here it's just `signingKey`?